src/lightmap.cpp

```diff
@@ -184,7 +180,7 @@ void map::generate_lightmap( const int zlev )
                     // When underground natural_light is 0, if this changes we need to revisit
                     // Only apply this whole thing if the player is inside,
                     // buildings will be shadowed when outside looking in.
-                    if (natural_light > LIGHT_SOURCE_BRIGHT && u_is_inside && !is_outside(x, y)) {
+                    if (natural_light > LIGHT_SOURCE_BRIGHT && !is_outside(x, y)) {
```

Comment:
`is_outside` now has to re-get the cache every time. It would be faster to use `map_cache.outside_cache` directly (or through a trivial getter).